src/containers/bookForm.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const BookForm = () => {
```

Comment:
`BookForm` should be a stateful component and its state should be initialized with `title` and `category`.